EIPS/rentcollector.md

```diff
@@ -0,0 +1,73 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: Rent Collection for Blockchain Data
+    Author: Daniel A. Nagy <daniel@ethereum.org>, Zsolt Felfoldi <zsolt@ethereum.org>
+    Type: Standard Track
+    Category: Core
+    Status: Draft
+    Created: 2017-11-01
+
+
+## Simple Summary
+Introducing a way to collect rent from accounts that is proportional 
+to the cost that they impose on the network by storing a large state
+over a long period of time.
+
+## Abstract
+A new global parameter called `rent` similar to `gasLimit` that can be 
+modified by miner vote is introduced. It is initialized at 0 and is 
+assumed to have been zero before its introduction. The miner validating 
+a particular block can increase rent in a bounded way at the cost of 
+forfeiting a proportional part of the mining reward. Similarly, if 
+`rent` is more than zero, one can decrease it in a bounded way and 
+receive the same additional mining reward that would need to be 
+forfeited to raise it back.
+
+Each contract account gets a new balance called `lease` used for paying 
+rent that is initialized at 0 and is assumed to have been zero before 
+its introduction. For externally controlled accounts, their `lease` 
+balance is the same as their normal balance. With each new block, an 
+account is chosen in a deterministically random fashion with a 
+probability proportional to the amount of storage that they use. The 
+current value of `rent` is subtracted from their `lease` and if the 
```

Comment:
Does the rent get credited to the miner?

Reply:
Yes.

Reply:
So, if the rent is `X`, and the balance is `Y`, and `X` > `Y`, does the miner get `X` or `Y`? Creating new money or not?

Reply:
If `rent` exceeds `lease`, only `lease` is credited to the miner (no money created) and the account is removed.